buildSrc/src/main/kotlin/com/datadog/gradle/Dependencies.kt

```diff
@@ -168,6 +169,8 @@ object Dependencies {
             "com.github.bumptech.glide:okhttp3-integration:${Versions.Glide}"
         )
 
+        const val Room = "androidx.room:room-runtime:${Versions.Room}"
+        const val RoomKapt = "androidx.room:room-compiler:${Versions.Room}"
```

Comment:
This one should be defined in the `AnnotationProcessors` object below, and can simply be named `Room`